test/stubs/_includes/layouts/layout-cycle-b.njk

```diff
@@ -0,0 +1,6 @@
+---
+inherits: b
+secondinherits: b
+layout: layouts/layout-cycle-c.njk
+---
+{{ content | safe }} {{secondinherits}}
```

Comment:
Should I remove these front-matter variables from the new stubs since they're not getting used?